packages/api/src/controllers/taskController.js

```diff
@@ -270,6 +273,20 @@ const taskController = {
     const nonModifiable = getNonModifiable(typeOfTask);
     return async (req, res, next) => {
       try {
+        // Only to satisfy "isDeleted" function requirement
```

Comment:
If it's truly only for that and there is no other reason to open a transaction, I think you might be able to just pass `null` as the first argument to `baseController.isDeleted()` -- just tried that on another ticket and it seemed fine!

Reply:
@kathyavini Thanks for the suggestion.